modules/utils/claws/utils/tagged_data.hpp

```diff
@@ -31,52 +31,52 @@ namespace claws
     data_type data;
 
     constexpr tagged_data(tagged_data const &) noexcept = default;
-    constexpr tagged_data& operator=(tagged_data const &) noexcept = default;
+    constexpr tagged_data &operator=(tagged_data const &) noexcept = default;
 
     /// Construtor is explicit, to avoid conversion
     explicit constexpr tagged_data(data_type data = 0u) noexcept
       : data(data)
     {}
 
-#define CLAWS_TAGGED_DATA_PREFIX_OP(OP)				\
-    constexpr tagged_data &operator OP() noexcept		\
-    {								\
-      OP data;							\
-      return *this;						\
-    }
+#define CLAWS_TAGGED_DATA_PREFIX_OP(OP)                                                                                                                        \
```

Comment:
I'm not really convinced moving `\` so far to the right helps with readability

Reply:
Hmm, actually I used clang-format with the configuration file from this repository, but I agree they were moved quite far. I will update the configuration file to align the `\` as far left as possible